scripts/docker-build

```diff
@@ -34,7 +34,12 @@ if [ -n "$prefix" ]; then
 else
     img_slug="${repo}"
 fi
-img="openobservatory/$img_slug"
+
+if [ -z ${IMG_SLUG} ]; then
+    img="openobservatory/$img_slug"
+else
+    img="openobservatory/${IMG_SLUG}"
+fi
```

Comment:
Why do you need to override that?
The reason to codify that was to have explicit link between repository name and docker image name for readability, maintainability, grepability and all  other alike *abilities.

Reply:
Because `github.com/ooni/explorer` in the future will be `oo/explorer`, but currently is pushed to `oo/explorer-next` to avoid conflicting with the legacy explorer.

Reply:
So the story is that both `TheTorProject/ooni-explorer` and `ooni/explorer` become `openobservatory/explorer`, right?

Is it actually an issue? TBH, I'm 96% okay with both repositories building images with `openobservatory/explorer` name. Anyway, the deploy script uses tag and tag has date and commit hash, I don't expect any conflicts there.

If you still consider that an issue, can you rather specify exact name in the script instead of using some environment variable from some `.profile`?

One of core reasons to create this repo was to extract all the needed bits of knowledge from individual developers' laptops into shared repo and this `IMG_SLUG` override conflicts with that goal.

Reply:
> If you still consider that an issue, can you rather specify exact name in the script instead of using some environment variable from some .profile?

I don't think I understand what you mean by this. Does docker-build support passing a custom name for the IMG_SLUG?

Reply:
> Does docker-build support passing a custom name for the IMG_SLUG

No. And I think that it should not as IMG_SLUG value is not stored in this repo.

If you want to override image name for `ooni/explorer` repo, then, please, change the script to specify the name for that repo instead of introducing some external state to the script, e.g. parsing `git rev-parse --show-prefix` before it's stripped.

Reply:
You mean adding to this script a line similar to:

```
if [ $repo -eq "ooni/explorer"];then
img_slug="ooni/explorer-next"
fi
```

?

Reply:
Yep. So anyone running the script in the root of the repo will get approximately same results (modulo state of unpinned packages fetched from external repos).

Reply:
SGTM